source/Classroom/Basics/API/does_sendgrid_offer_a_sandbox_or_test_environment.md

```diff
@@ -1,24 +1,27 @@
 ---
 seo:
   title: Does Sendgrid offer a sandbox or test environment?
-  description: Does Sendgrid offer a sandbox or test environment?
-  keywords: box, throughput, development, dev, staging, sand, speed, developers, test, sandbox, environment, trial
+  description: Learn about options to help test your integrations...
+  keywords: sandbox, sand, box, throughput, development, dev, staging, speed, developers, test, sandbox, environment, trial
 title: Does Sendgrid offer a sandbox or test environment?
 weight: 0
 layout: page
-zendesk_id: 203891358
 navigation:
   show: true
 ---
 
 We do not offer a true "sandbox" environment, however we do have a [free trial](https://sendgrid.com/transactional-email/pricing) which allows you to send 100 emails per day, and also gives you access to almost all of the features offered with our higher package levels. The free trial does not include access to a dedicated IP or IP whitelabeling.
 
-We also have a [sink email address]({{root_url}}/Classroom/Troubleshooting/Delivery_Issues/safely_test_your_sending_speed.html) that can help you test message throughput for your integration.
+If you are looking to test API integration, you can enable the sandbox_mode in the [mail_settings parameters of our Mail Send API](https://sendgrid.com/docs/Classroom/Send/v3_Mail_Send/sandbox_mode.html). **This will allow you to validate each parameter you include, and the structure of your JSON payload, all without actually sending a message.**
```

Comment:
We merged in a pull request from an outside user that adds in the first part (which is why there's a conflict now). So when you combine the existing requests, you no longer need to update this section.